main_app/models.py

```diff
@@ -7,6 +7,7 @@
 class Category(models.Model):
     name = models.CharField(max_length=100)
     description = models.TextField(max_length=250)
+    is_deleted = models.BooleanField(default=False)
```

Comment:
This is_deleted boolean field is in preparation for the reworking of the delete function